### ISSUE TITLE
Build a one page download site for the app on the gh-pages branch

### ISSUE THREAD

**Issue by mariobehling** [fossasia/loklak_wok_android#11]
Build a one page download site for the app on the gh-pages branch.

**Comment by SidharthBansal** [comment]
Can I work on this .Please provide the download link

**Comment by abishekvashok** [comment]
@SidharthBansal which download link?

**Comment by SidharthBansal** [comment]
link for the app from where it has to downloaded

**Comment by abishekvashok** [comment]
The apk should be uploaded to gh-pages branch via Travis, see the phimpme-android repo for an example

**Comment by abishekvashok** [comment]
@mariobehling how does this look?
![screenshot from 2017-06-02 20-24-23](https://cloud.githubusercontent.com/assets/8947010/26731386/92e6a8f4-47d1-11e7-91b0-fa49713988e0.png)

Should I make a PR?

**Comment by mariobehling** [comment]
Yes, this download site should go to wok.loklak.net into the gh-pages branch of this repository.

**Comment by abishekvashok** [comment]
@mariobehling please create a `gh-pages` branch from the `apk` branch then, so that I can make a pull request.

Working Demo: https://abishekvashok.github.io/loklak_wok_android/

*Note: The download button link will only works after #25 is merged.*

**Comment by abishekvashok** [comment]
^^ @mariobehling I can't create branches so pls do so.

**Comment by abishekvashok** [comment]
@mariobehling are we doing this issue now, I meant priority? As i have finished this up back in the past.

**Comment by mariobehling** [comment]
Please use the Readme.md and yaydoc to create the page in the gh-pages branch.
